01-Git_Bash_Make/Task2_bash_script/script

```diff
@@ -2,5 +2,12 @@
 
 directory=/tmp/guesanumber
 
-cp -R /home/igor/gl-kernel-basecamp-2019/01-Git_Bash_Make/Task1_game $directory
+if ! [ -d $directory ]
```

Comment:
IMO it's preferable to keep all the condition expession (including inversion) within test statement.
And as I've already said - keep string arguments in quotes (`"${VAR}"`).